packages/ruleset/test/utils/root-document.js

```diff
@@ -0,0 +1,137 @@
+module.exports = {
+  openapi: '3.0.0',
```

Comment:
This is a clean API definition that I use for the tests. Easier to edit and make one thing go wrong than it is to write new definitions a bunch